### PR TITLE
Unify losses that have target parameters

### PR DESCRIPTION
**Pull request by 22quinn**
Closes https://github.com/facebookresearch/rl/issues/65

### CODE REVIEW

**Comment by 22quinn** [comment]
Looks like my own circleci does not have enough resources to run the unit tests. Is it possible for repo maintainers to trigger CI for this PR or enable this option: https://circleci.com/docs/2.0/oss/#build-pull-requests-from-forked-repositories?

File: torchrl/objectives/costs/ddpg.py [hunk 1]
```diff
@@ -30,19 +30,24 @@ class DDPGLoss(_LossModule):
         device (str, int or torch.device, optional): a device where the losses will be computed, if it can't be found
             via the value operator.
         loss_function (str): loss function for the value discrepancy. Can be one of "l1", "l2" or "smooth_l1".
+        delay_actor (bool): whether to separate the target actor networks from the actor networks used for data
+            collection.
+        delay_value (bool): whether to separate the target value networks from the value networks used for data
```
Comment (vmoens) [inline torchrl/objectives/costs/ddpg.py:35]: I know I did not do it for the other optional params, but perhaps you could add `(bool, optional)` and indicate the default value?

File: torchrl/objectives/costs/dqn.py [hunk 2]
```diff
@@ -30,19 +28,20 @@ class DQNLoss(_LossModule):
         value_network (ProbabilisticTDModule): a Q value operator.
         gamma (scalar): a discount factor for return computation.
         loss_function (str): loss function for the value discrepancy. Can be one of "l1", "l2" or "smooth_l1".
+        delay_value (bool): whether to duplicate the value network into a new target value network to create double DQN
```
Comment (vmoens) [inline torchrl/objectives/costs/dqn.py:31]: Same as above

**Review by vmoens — APPROVED** [review]
This is truly brilliant! Thanks so much for this.
Perhaps could you fix the minor comment I left regarding the docstrings, then we're good to merge!

File: torchrl/objectives/costs/ddpg.py (hunk 1 above)
Comment (22quinn) [reply torchrl/objectives/costs/ddpg.py]: Sure, these are now added in the latest commit.

File: torchrl/objectives/costs/dqn.py (hunk 2 above)
Comment (22quinn) [reply torchrl/objectives/costs/dqn.py]: Added. Thanks for reviewing!

**Pull request merged**